scripts/qa_crowbarsetup.sh

```diff
@@ -1233,12 +1241,23 @@ function create_repos_yml()
         SUSE-Enterprise-Storage-2-Updates-test=http://$distsuse/ibs/SUSE:/Maintenance:/Test:/Storage:/2:/x86_64/update/ \
         >> $tmp_yml
```

Comment:
Some comments below you mention there's a wrong code with a test for cloud7 and milestone tags that deals with SLES 12 GA. Should that be removed there too?

https://github.com/SUSE-Cloud/automation/pull/991/files/8450c45b6376fabb21c11f18201340208db5cbfc#diff-c202af9895fd7be9224b15a93a00b0ffR1930